config/crd/bases/perses.dev_perses.yaml

```diff
@@ -291,9 +291,6 @@ spec:
                           disable_custom:
                             type: boolean
                           options:
-                            items:
-                              format: duration
-                              type: string
```

Comment:
@jgbernalp, As you can see, `make manifests` removes this piece of code. 
 [this is the annotations we have for this field](https://github.com/perses/perses/blob/3d34cf1b8849a8bf648e0d455d59b8bb32362c31/pkg/model/api/config/frontend.go#L40-L43)